db/kv_dense_channel_storage_reader.go

```diff
@@ -0,0 +1,221 @@
+//  Copyright (c) 2016 Couchbase, Inc.
+//  Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file
+//  except in compliance with the License. You may obtain a copy of the License at
+//    http://www.apache.org/licenses/LICENSE-2.0
+//  Unless required by applicable law or agreed to in writing, software distributed under the
+//  License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+//  either express or implied. See the License for the specific language governing permissions
+//  and limitations under the License.
+
```

Comment:
I guess it makes sense to leave the copyright license here, but can it be shortened to this?

```
Copyright (c) 2016 Couchbase, Inc.
Licensed under the Apache License, Version 2.0.  See LICENSE for details.
```